force-app/main/default/pages/lmsPublisherVisualforce.page

```diff
@@ -0,0 +1,40 @@
+<apex:page controller="LMSVisualforceController" docType="html-5.0">
+    <!-- <apex:includeScript value="{!URLFOR($Resource.lmsvf, 'lmsvf.js')}"/> -->
+    <apex:slds />
+    <div class="slds-card">
+        <div class="slds-card__body slds-card__body_inner">
+            <!-- Placeholder to build list. -->
+            <ul data-list="list"></ul>
+        </div>
+
+        <footer class="slds-card__footer">
+            <c:viewSource
+                source="pages/{!$CurrentPage.Name}.page"
+                branch="beta"
+            >
+                Publish a Lightning Message Service message for other
+                Visualforce pages, LWCs, and Aura components to respond to. This
+                page publishes the <strong>Record_Selected__c</strong> event.
+                Click a contact in the list to notify subscribers on this page.
+                The selected contact detail should then appear in the
+                subscribers.
+            </c:viewSource>
+        </footer>
+    </div>
+
+    <apex:remoteObjects>
+        <apex:remoteObjectModel name="Contact" fields="Id,Name,Picture__c" />
+    </apex:remoteObjects>
+
+    <!-- prettier-ignore -->
```

Comment:
We can probably drop that if it's just for keeping blank lines within the `script` tag